### PR TITLE
Basic networking automation test

### PR DESCRIPTION
**Pull request by AMZN-scspaldi**
This is a simpler version of the existing networking test that Gene worked on, intended to give us more specific testing feedback.

### CODE REVIEW

File: AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_BasicConnectivity_Connects.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+# Test Case Title : Check that the basic client connect test works
+
+
+# fmt: off
+class Tests():
```
Comment (AMZN-Gene) [inline AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_BasicConnectivity_Connects.py:13]: Are all these variables used? Might delete the ones we dont need, for example "Tests.found_unexpected_lines " is never used.

**Review by AMZN-Gene — APPROVED** [review]

File: AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_BasicConnectivity_Connects.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
```
Comment (lmbr-pip) [inline AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_BasicConnectivity_Connects.py:13]: nit: class Tests seems incorrectly named, "Tests" is generic and it doesn't contain tests but constant strings. Consider TestConstants or similar

File: AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_BasicConnectivity_Connects.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    enter_game_mode =           ("Entered game mode",                       "Failed to enter game mode")
+    exit_game_mode =            ("Exited game mode",                        "Couldn't exit game mode")
+    find_network_player =       ("Found network player",                    "Couldn't find network player")
+# fmt: on
+
+
+def Multiplayer_BasicConnectivity_Connects():
+    r"""
+    Summary:
+    Runs a test to make sure that a player can be spawned
```
Comment (lmbr-pip) [inline AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_BasicConnectivity_Connects.py:23]: nit: .. a networked player can be spawned.

File: AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_BasicConnectivity_Connects.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    :return:
+    """
+    import azlmbr.legacy.general as general
+    from editor_python_test_tools.utils import Report
+    from editor_python_test_tools.utils import Tracer
+
+    from editor_python_test_tools.utils import TestHelper as helper
+
+
+    level_name = "BasicConnectivity_Connects"
```
Comment (lmbr-pip) [inline AutomatedTesting/Gem/PythonTests/Multiplayer/tests/Multiplayer_BasicConnectivity_Connects.py:43]: nit: If var is constant I would use either use ALL_CAPS or use : Final (if we support Py 3.8) to indicate it should be not be reassigned.

**Review by lmbr-pip — COMMENTED** [review]
LG.

**Review by AMZN-puvvadar — APPROVED** [review]

**Pull request merged**